src/types/step.d.ts

```diff
@@ -216,6 +216,12 @@ declare namespace Step {
      * The HTML text of the button
      */
     text?: string;
+
+    /**
+     * Whether the button should be disabled
+     * When the value is `true` the button will be disabled
+     */
+    disabled?: boolean;
```

Comment:
Didn't we want to allow passing functions for disabled as well?

Reply:
I thought about it but was not sure when we should re-evaluate the function.  
When do you think is suitable? On every show event? On an interval?

Reply:
I would think we would do the same thing we do with `text`.